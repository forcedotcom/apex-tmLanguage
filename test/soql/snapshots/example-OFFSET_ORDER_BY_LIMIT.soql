SELECT Name, Id FROM Merchandise__c ORDER BY Name LIMIT 20 OFFSET 100
