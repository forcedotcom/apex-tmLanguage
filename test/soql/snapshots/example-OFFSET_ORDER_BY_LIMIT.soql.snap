>SELECT Name, Id FROM Merchandise__c ORDER BY Name LIMIT 20 OFFSET 100
#^^^^^^ source.soql keyword.operator.query.select.apex
#      ^ source.soql
#       ^^^^ source.soql keyword.query.field.apex
#           ^ source.soql punctuation.separator.comma.apex
#            ^ source.soql
#             ^^ source.soql keyword.query.field.apex
#               ^ source.soql
#                ^^^^ source.soql keyword.operator.query.from.apex
#                    ^ source.soql
#                     ^^^^^^^^^^^^^^ source.soql storage.type.apex
#                                   ^ source.soql
#                                    ^^^^^^^^ source.soql keyword.operator.query.orderby.apex
#                                            ^ source.soql
#                                             ^^^^ source.soql keyword.query.field.apex
#                                                 ^ source.soql
#                                                  ^^^^^ source.soql keyword.operator.query.apex
#                                                       ^ source.soql
#                                                        ^^ source.soql constant.numeric.decimal.apex
#                                                          ^ source.soql
#                                                           ^^^^^^ source.soql keyword.operator.query.apex
#                                                                 ^ source.soql
#                                                                  ^^^ source.soql constant.numeric.decimal.apex
>